src/translations.js

```diff
@@ -0,0 +1,129 @@
+export const resources = {
+  en: {
+    common: {
+      openChat: 'Open Chat',
```

Comment:
This flat list of keys is not really scaleable. I would love to see these grouped and nested into a reasonable hierarchy.